flake8_future_import.py

```diff
@@ -150,7 +178,7 @@ def main(args):
         ignored = set()
     for filename in args.files:
         with open(filename, 'rb') as f:
-            tree = compile(f.read(), filename, 'exec', PyCF_ONLY_AST)
+            tree = ast.parse(f.read(), filename=filename, mode='exec')
```

Comment:
the `from __future__ import print_function` was leaking into compiled code

https://docs.python.org/2/library/functions.html#compile
> The optional arguments flags and dont_inherit control which future statements (see PEP 236) affect the compilation of source. If neither is present (or both are zero) the code is compiled with those future statements that are in effect in the code that is calling compile().

That's solvable, but I just changed this to `ast.parse` because that's what the docs recommend.